lib/vet360/service.rb

```diff
@@ -50,7 +50,11 @@ def log_error_messages(error)
           vet360: 'failed_vet360_id_initializations'
         )
       else
-        log_message_to_sentry(error.message, :error, extra_context: { url: config.base_path, body: error.body })
+        log_message_to_sentry(
+          error.message,
+          :error,
+          url: config.base_path, body: error.body
```

Comment:
I mentioned this in a previous comment, but rubocop balks:

```
 Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.
```

Reply:
Gotcha.  This actually works in our favor.  Let's add in a unique, specific [tag k/v pair into the fourth param](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/vet360/service.rb#L50).  This will both satisfy RuboCop, and will make it **much** easier to grep for this error in Sentry.  

Perhaps something like `vet360: 'general_client_error'`, or something to that effect?

Reply:
I like this option. Updated the specs to be a bit more specific as well.